tools/dev/k3d/Makefile

```diff
@@ -28,21 +38,24 @@ create-registry:
 		echo "Registry already exists"; \
 	fi
 
-create-cluster:
-	@if ! k3d cluster list | grep -q -m 1 $(CLUSTER_NAME); then \
-		echo "Creating cluster"; \
-		k3d cluster create $(CLUSTER_NAME) \
-			--servers 1 \
-			--agents 3 \
-			--registry-use "k3d-grafana:$(REGISTRY_PORT)" \
-			--wait; \
-	else \
-		echo "Cluster already exists"; \
-	fi
+secrets: secrets/grafana.jwt secrets/gel.jwt
 
-prepare: create-registry create-cluster update-repos build-latest-image
+secrets/grafana.jwt:
+	mkdir -p secrets/
+	op document get "loki/grafana.jwt" > $(CURDIR)/secrets/grafana.jwt
```

Comment:
Can you add a note or a check somewhere that this is the command line version of 1Password? There is a op tool in Linux (https://linux.die.net/man/1/op) as well.

Reply:
Ahh, I didn't know about that. Yes, good call, will do! And for anyone reading this, those secrets are in the shared grafana 1password.